docs/book/architecture.rst

```diff
@@ -17,6 +17,11 @@ Resource Layer
 We created an abstraction on top of Doctrine, in order to have a consistent and flexible way to manage all the resources. By "resource" we understand every model in the application.
 Simplest examples of Sylius resources are "product", "order", "tax_category", "promotion", "user", "shipping_method" and so on...
 
+There are two types of resources in **Sylius**:
+
+* registered by default - their names begin with ``sylius_*`` for example: ``sylius_product``
+* custom resources, from your application which have a separate convention. We place them under ``sylius_resource:`` ``resource_name:`` in the ``config.yml``.
```

Comment:
We should add a note, that we recommend to use an `app.*` convention with custom resources.